src/images/static-patron-code-flow.svg

```diff
@@ -1 +1 @@
f" font-size="8.8pt" font-style="normal" font-weight="normal" text-decoration="normal" x="590.1599458639388" y="750.1125355359999" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">of Authorization</text></g><path fill="white" stroke="black" paint-order="fill stroke markers" d=" M 117.64580033909463 793.5641763139998 L 117.64580033909463 793.5641763139998 A 15.832581659999999 15.832581659999999 0 0 1 133.47838199909464 777.7315946539999 L 302.83681248716834 777.7315946539999 L 302.83681248716834 777.7315946539999 A 15.832581659999999 15.832581659999999 0 0 1 318.66939414716836 793.5641763139998 L 318.66939414716836 802.360055014 L 318.66939414716836 802.360055014 A 15.832581659999999 15.832581659999999 0 0 1 302.83681248716834 818.1926366739999 L 133.47838199909464 818.1926366739999 L 133.47838199909464 818.1926366739999 A 15.832581659999999 15.832581659999999 0 0 1 117.64580033909463 802.360055014 L 117.64580033909463 793.5641763139998" stroke-miterlimit="10" stroke-width="1.4659797833333332" stroke-dasharray=""/><g><text fill="green" stroke="none" font-family="sans-serif" font-size="11pt" font-style="normal" font-weight="normal" text-decoration="normal" x="142.27426069909464" y="802.360055014" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">✔</text><text fill="black" stroke="none" font-family="sans-serif" font-size="11pt" font-style="normal" font-weight="normal" text-decoration="normal" x="154.69092768365275" y="802.360055014" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve"> Display Confirmation</text></g><g><g/><path fill="none" stroke="black" paint-order="fill stroke markers" d=" M 17.591757400000006 543.233468512 L 690.750949047474 543.233468512 L 690.750949047474 697.8650160579999 L 17.591757400000006 697.8650160579999 L 17.591757400000006 543.233468512 Z" stroke-miterlimit="10" stroke-width="2.5131082" stroke-dasharray=""/><path fill="white" stroke="black" paint-order="fill stroke markers" d=" M 17.591757400000006 543.233468512 L 17.591757400000006 564.3435773919999 L 117.30860502470703 564.3435773919999 L 127.86365946470704 553.788522952 L 127.86365946470704 543.233468512 L 17.591757400000006 543.233468512" stroke-miterlimit="10" stroke-width="2.5131082" stroke-dasharray=""/><text fill="black" stroke="none" font-family="sans-serif" font-size="8.8pt" font-style="normal" font-weight="bold" text-decoration="normal" x="35.183514800000005" y="557.306874432" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">Conditional</text><g><rect fill="white" stroke="none" x="142.81665325470703" y="544.640809104" width="173.5775302717578" height="18.295427696"/></g><text fill="black" stroke="none" font-family="sans-serif" font-size="8.8pt" font-style="normal" font-weight="bold" text-decoration="normal" x="145.45541686470705" y="557.306874432" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">[Centrapay Triggers ID Check]</text></g></g></g></svg>
```

Comment:
The create payment request call in this diagram has a `barcodeType` parameter. Does this exist/will it exist in the future?